### PR TITLE
[del class chans] adds delclasschans cmd, ignores admin cmds in help

### PR DESCRIPTION
**Pull request by c650**
#77

### CODE REVIEW

File: src/app/plugins/delclasschans.plugin.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  public async execute(message: IMessage, args: string[]) {
+    const channels = this.container.classService.getClasses(ClassType.ALL);
+    const numChannels = channels.size;
+
+    const deleteCaller = message.author.tag;
+
+    await message.reply(`Deleting ${numChannels} channels at request of ${deleteCaller}`);
+
+    channels.forEach((channel, _) => {
+      console.log(`Channel ${channel.name} deleted by ${deleteCaller}`);
```
Comment (joey-colon) [inline src/app/plugins/delclasschans.plugin.ts]: remove

File: src/app/plugins/delclasschans.plugin.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import { Plugin } from '../../common/plugin';
+import { IContainer, IMessage, ChannelType, ClassType } from '../../common/types';
+
+export class DeleteClassChannelsPlugin extends Plugin {
+  public name: string = ' Delete classes';
```
Comment (joey-colon) [inline src/app/plugins/delclasschans.plugin.ts]: remove leading space

File: src/app/plugins/delclasschans.plugin.ts [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    return (
+      args.join(' ') ===
+      'Yes I understand that this is a really really serious thing to do and I will not be stupid and do it prematurely or maliciously.'
+    );
+  }
+
+  public async execute(message: IMessage, args: string[]) {
+    const channels = this.container.classService.getClasses(ClassType.ALL);
+    const numChannels = channels.size;
+
```
Comment (joey-colon) [inline src/app/plugins/delclasschans.plugin.ts]: nit: remove line of whitespace

File: src/app/plugins/delclasschans.plugin.ts [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  public permission: ChannelType = ChannelType.Admin;
+
+  constructor(public container: IContainer) {
+    super();
+  }
+
+  public validate(message: IMessage, args: string[]) {
+    return (
+      args.join(' ') ===
+      'Yes I understand that this is a really really serious thing to do and I will not be stupid and do it prematurely or maliciously.'
```
Comment (joey-colon) [inline src/app/plugins/delclasschans.plugin.ts]: Still debating if it's even worth to include this ...

File: src/app/plugins/delclasschans.plugin.ts (hunk 1 above)
Comment (c650) [reply src/app/plugins/delclasschans.plugin.ts]: there's no other accountability for who requested teh deletion

File: src/app/plugins/delclasschans.plugin.ts (hunk 4 above)
Comment (c650) [reply src/app/plugins/delclasschans.plugin.ts]: let's not, but also just not deploy this patch till it's time to delete, and then we can disable the plugin till next end of sem

**Pull request merged**